### PR TITLE
Fix error when VM is stopped gracefully

### PR DESCRIPTION
**Pull request by ernierasta**
This is trivial fix to get rid of error when VM stops gracefully.

Without fix it looks like:

 * Stopping VM mail ...
   Waiting 40 seconds for VM shutdown ........sh: can't kill pid 17659: No such process    [ ok ]

### CODE REVIEW

File: qemu.initd [hunk 1]
```diff
@@ -205,7 +205,7 @@ version() {
 #-------------------------------- Helpers -------------------------------
 
 is_running() {
-	[ -e "$pidfile" ] && kill -0 "$(cat "$pidfile")"
+	[ -e "$pidfile" ] && kill -0 "$(cat "$pidfile")" &> /dev/null
```
Comment (jirutka) [inline qemu.initd]: This is bashism, POSIX does not define `&>`; `>/dev/null 2>&1` should be used instead.

Comment (jirutka) [inline qemu.initd]: Actually only `2>/dev/null` is needed here.

**Comment by jirutka** [comment]
I’ve ported your fix in 1b83390b83ce7c33aa64a53758ee7f6748b5d62a. Thanks!

File: qemu.initd (hunk 1 above)
Comment (ernierasta) [inline qemu.initd]: You are absolutely right. ;-)

**Pull request closed without merging**